src/test/java/org/zalando/fahrschein/redis/RedisCursorManagerIT.java

```diff
@@ -0,0 +1,66 @@
+package org.zalando.fahrschein.redis;
+
+import org.junit.Ignore;
+import org.junit.Test;
+import org.springframework.data.redis.connection.jedis.JedisConnectionFactory;
+import org.zalando.fahrschein.CursorManager;
+import org.zalando.fahrschein.domain.Cursor;
+import redis.clients.jedis.JedisShardInfo;
+
+import java.io.IOException;
+import java.util.Collection;
+
+import static org.hamcrest.collection.IsEmptyCollection.empty;
+import static org.hamcrest.collection.IsIterableContainingInAnyOrder.containsInAnyOrder;
+import static org.junit.Assert.assertThat;
+
+public class RedisCursorManagerIT {
+
+    public static final String EVENT_TYPE_NAME = generateUniqueEventType();
+
+    @Test
+    @Ignore("Meant for local testing. You need a running redis cluster on localhost.")
```

Comment:
How easy is it to start a local redis instance, for example in a docker container?

I tried to automate this for postgres inside the mvn pre-/post-integration-test phases using a docker-maven-plugin (https://fabric8io.github.io/docker-maven-plugin/), but could not connect from the test to the db. Starting the same docker image manually worked.